uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/rules/InstallBrokerTestRule.java

```diff
@@ -52,6 +54,13 @@ public Statement apply(final Statement base, final Description description) {
             public void evaluate() throws Throwable {
                 Logger.i(TAG, "Applying rule....");
                 Logger.i(TAG, "Installing broker: " + ((App) broker).getAppName());
+
+                if (BuildConfig.PRE_INSTALL_LTW) {
+                    final BrokerLTW brokerLTW = new BrokerLTW();
```

Comment:
can we move this RulesHelper.java like this? https://github.com/AzureAD/microsoft-authentication-library-common-for-android/blob/a2b6d6423079de31f570c9db27bcfd8e0c8dd75f/uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/rules/RulesHelper.java#L88

Reply:
Also, do we know if PREFER_PRE_INSTALLED_APKS is being set currently in the pipelines?
Looks like we have to add BrokerLTW in the preinstalled apks rule as well

Reply:
LTW is covered already in the remove pre installed brokers rule
I'll mention the flag in the description